pkg/scheduler/plugins/drf/drf.go

```diff
@@ -107,7 +116,7 @@ func (drf *drfPlugin) OnSessionOpen(ssn *framework.Session) {
 			}
 		}
 
-		glog.V(4).Infof("Victims from DRF plugins are %+v", victims)
+		glog.V(3).Infof("Victims from DRF plugins are %+v", victims)
```

Comment:
let keep log level to 4.

Reply:
OK.  Currently the same log level is inconsistent in different plugin. For example, in Gang plugin the following log is Level3. I will change it to Level4 as well.

> glog.V(3).Infof("Victims from Gang plugins are %+v", victims)